docs/units/equivalencies.rst

```diff
@@ -415,6 +415,9 @@ between the Celsius, Fahrenheit, and Kelvin. For example::
     >>> temp_F = temp_C.to(u.imperial.deg_F, equivalencies=u.temperature())
     >>> temp_F  # doctest: +FLOAT_CMP
     <Quantity 32. deg_F>
+    >>> temp_F = temp_C.to(u.imperial.deg_R, equivalencies=u.temperature())
```

Comment:
Oops, this probably should be `temp_R`, no?

Reply:
Oh sorry 😅